awesome_classifier.py

```diff
@@ -0,0 +1,35 @@
+from torchvision import models
+import torch
+dir(models)
+
+alexnet = models.alexnet(pretrained=True)
+
```

Comment:
For python scripts, it is a good practice to put all import statements at the beginning of the script